README.md

```diff
@@ -1,6 +1,6 @@
 # Cylc TextMate Bundle
 
-Last updated: 2020-06-26
+Last updated: <span actions:bind='update-date'>2020-06-26</span>
```

Comment:
Might as well put this in too. Should prevent anyone accidentally breaking the updating of this date

Reply:
Meh, I think it was safe enough before but this is fine too.

(FYI `actions:bind` is an illegal attribute)

Reply:
Well, the browser strips `actions:bind` out, which is fine

Reply:
If you prefix it with `data-` then it's valid HTML.

Reply:
Trouble with `id` is it isn't clear this is a sort of data binding used by Actions. Could use `data-actions-bind` but it's extra characters :P